config/env.go

```diff
@@ -0,0 +1,131 @@
+package config
+
+import (
+	"crypto/tls"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// The following variables contain the names of environment variables that can
+// be set to provide configuration values.
+const (
+	envVarAuthURL = "HCP_AUTH_URL"
+
+	envVarClientID     = "HCP_CLIENT_ID"
+	envVarClientSecret = "HCP_CLIENT_SECRET"
+
+	envVarPortalURL = "HCP_PORTAL_URL"
+
+	envVarAPIAddress = "HCP_API_ADDRESS"
+
+	// envVarAPIHostnameLegacy is necessary as `HCP_API_HOST` has been used in
+	// the past and needs to be supported for backward compatibility.
+	envVarAPIHostnameLegacy = "HCP_API_HOST"
```

Comment:
There is none. `HCP_API_ADDRESS` supersedes `HCP_API_HOST`. 

I find address to be the more fitting name, as it can include the port. Especially for the `SCADA` variant. It would be an option to keep using `HCP_API_HOST` and only introduce the `SCADA` variable. I would however prefer the consistency.